models/stock_picking.py

```diff
@@ -61,20 +61,28 @@ def is_compatible_package(self, package_name):
 
         return res
 
-    def update_picking(self, expected_package_name=None, **kwargs):
+    def update_picking(self, expected_package_name=None, u_transport_id=None, **kwargs):
         """ Extend update_picking with a new parameter expected_package_name
             to be used during swapping packages
             TODO: finish implement and add parameter at README
         """
+        # if picking_info already exists in kwargs, update with trailer info
```

Comment:
Should be in the kwargs,not a separate param

Reply:
moved to kwargs

Reply:
This is being handled in branch udes-story/528